### PR TITLE
prevent image requests and downloads, empty tweets, and undefined image source

### PR DESCRIPTION
**Pull request by MrKarlDilkington**
https://github.com/jasonmayes/Twitter-Post-Fetcher/issues/95

### CODE REVIEW

File: js/twitterFetcher.js [hunk 1]
```diff
@@ -179,11 +179,25 @@
...
               '&callback=twitterFetcher.callback&' +
               'suppress_response_codes=true&rnd=' + Math.random();
-        } 
+        }
         head.appendChild(script);
       }
     },
     callback: function(data) {
+      // data.body is the pre-parsed HTML that can be searched for img elements to remove
+      // console.log(data.body);
```
Comment (jasonmayes) [inline js/twitterFetcher.js]: Please remove the comments here it is clear from the code what is being done.

**Comment by jasonmayes** [comment]
Thanks for this! 1 minor comment but if you could just tidy those comments up I can then merge. Cheers!

**Comment by MrKarlDilkington** [comment]
@jasonmayes 

I made a second commit that removed all the comments, except for the three relating to the regex.

**Comment by jasonmayes** [comment]
Thank you for contributing!

**Pull request merged**